jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/response/JaxRsResultFactory.java

```diff
@@ -80,10 +101,10 @@ public JaxRsResult<T> create(Flux<T> output, Object[] args) {
         return result;
     }
 
-    private Func1<Flux<T>, Flux<byte[]>> createSerializer(String type, Class<T> dataCls, boolean isSingleType, JsonSerializerFactory jsonSerializerFactory) {
+    private Func1<Flux<T>, Flux<byte[]>> createStreamSerializer(String type, Class<T> dataCls, Type streamType, JsonSerializerFactory jsonSerializerFactory) {
```

Comment:
We have two JaxRsResultFactories. The one we have here and also the one that is named [JaxRsStreamingResultFactory](https://github.com/FortnoxAB/reactive-wizard/blob/master/jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/response/JaxRsStreamingResultFactory.java). Why do we have the code that manage the streaming in the parent class rather in the child class with the naming "streaming"?